actpool/actpool.go

```diff
@@ -141,9 +149,24 @@ func NewActPool(g genesis.Genesis, sf protocol.StateReader, cfg Config, opts ...
 		return nil, err
 	}
 	ap.timerFactory = timerFactory
+
+	for i := 0; i < _numWorker; i++ {
+		ap.jobQueue[i] = make(chan workerJob, ap.cfg.MaxNumActsPerAcct)
+		ap.worker[i] = newQueueWorker(ap, ap.jobQueue[i])
+		if err := ap.worker[i].Start(); err != nil {
+			return nil, err
+		}
+	}
 	return ap, nil
 }
 
+// TODO: add start() and stop() in actpool
+// func (ap *actPool) Start() {
+// }
+
+// func (ap *actPool) Stop() {
+// }
+
```

Comment:
remove if not needed

Reply:
or should put `ap.worker[i].Start()` at L156 here?

Reply:
add as a todo task